pyvista/utilities/docs.py

```diff
@@ -25,6 +25,7 @@ def copy_function_doc(source, alias=False, deprecated=False):
     def wrapper(func):
         if source.__doc__ is None or len(source.__doc__) == 0:
             raise ValueError('Cannot copy docstring: docstring was empty.')
+        # issue is here:
         doc = source.__doc__
```

Comment:
```suggestion
        doc = '\n'.join([ln.strip() for ln in source.__doc__.splitlines()])
```

@GuillaumeFavelier, if we change it to that ^^, then all the formatting issues are fixed, but the comparison in the tests fail

Reply:
Actually, I'm wrong - this doesn't fix the formatting as it will destroy any wanted indentations as well like descriptions of Parameters